hypemaths/models/matrix.py

```diff
@@ -30,6 +33,80 @@ def dims(self) -> tuple:
     def __repr__(self) -> str:
         return f"{self.__class__.__name__}({self.matrix})"
 
+    def multiply_by_number(self, number: t.Union[int, float]) -> 'Matrix':
+        '''
+        Multiplies the current matrix with a number.
+        '''
```

Comment:
```suggestion
    def multiply_by_number(self, number: t.Union[int, float]) -> "Matrix":
        """Multiplies the current matrix with a number.""" 
```
Let's use `"` as the standard quotes as it's more universal and also `'` is used as secondary ones 😁